sharkfuser/include/fusili/graph.h

```diff
@@ -143,6 +149,39 @@ Graph::convFProp(const std::shared_ptr<TensorAttr> &x,
   return y;
 }
 
+inline ErrorOr<std::string> Graph::emitVMFB() {
```

Comment:
Please add lit/integration tests for this?

Reply:
What are you thinking here? I added a test to [sharkfuser/samples/convolution/conv_fprop.cpp](https://github.com/nod-ai/shark-ai/pull/1936/files#diff-eafc10905e11974cc5b33d82d295e9db72b3962bb25991d84efd36236e960193).

I was thinking the emit .vmfb logic would mainly be tested through e2e integration tests (that we can't write until the runtime is hooked up). Lit tests checking the exact vmfb would break every time we upgrade IREE. There's probably a magic couple of bytes as part of the header for any vmfb, I could check for that in a standard test.